src/misc/announcements.ts

```diff
@@ -32,3 +34,25 @@ setInterval(() => {
   index++;
   index = index % MESSAGES_LENGTH;
 }, INTERVAL_MINUTES * 60 * 1000);
+
+/**
+ * Broadcast the localized string to all players
+ */
+export function announceTranslation(translationKey: string, ...args: string[]) {
```

Comment:
```suggestion
export function announce(translationKey: string, ...args: string[]) {
```

Since we *should* translate all announcements/broadcasts, maybe something shorter?